core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaRegistry.java

```diff
@@ -33,6 +34,8 @@
 
   SchemaString get(int id) throws SchemaRegistryException;
 
+  boolean hasSubjectConfig(String subject) throws SchemaRegistryStoreException;
```

Comment:
I'm not wild about adding this to the SchemaRegistry interface.

This is a public interface, so we should be a little bit careful about adding new methods willy-nilly. Also, the methods here closely parallel the actual rest api, and this new method has no direct analog in the rest api.